CONTRIBUTING.md

```diff
@@ -42,6 +42,32 @@ pip install -r requirements-dev.txt
    <code> tox -eblack </code>
 - To Fix the black violation  <code> black <PATH_FILE_YOU_WANT_TO_FIX> </code>
 
+# Create a preview of the website from your fork
```

Comment:
Hm, how do you preview the site locally? I don't think this is necessary to say how to preview from the fork. Most people prefer to iterate locally rather than through CI since it's much faster.

Reply:
Okay! I changed the instructions to how to preview the website locally.